Packages/com.unity.inputsystem/InputSystem/Plugins/Android/AndroidGameController.cs

```diff
@@ -223,7 +231,7 @@ public class DualShock4GamepadAndroid : DualShockGamepad
     }
 
     /// <summary>
-    /// A PlayStation DualShock 4 controller connected to an Android device.
+    /// A XboxOne controller connected to an Android device.
```

Comment:
```suggestion
    /// An XboxOne controller connected to an Android device.
```

Reply:
(Because despite being a consonant, X produces a sound starting with `e`).

Reply:
You are right!